Gemfile

```diff
@@ -2,3 +2,6 @@ source 'https://rubygems.org'
 
 gem 'asciidoctor'
 gem 'sass'
+
+# For TOC generation
+gem 'nokogiri'
```

Comment:
Please fix the line ending:

![image](https://github.com/theforeman/foreman-documentation/assets/12595287/dec1661e-ffeb-4723-9512-4c59f01b1388)